### DISCUSSION TITLE
Adding additional "reference" sequences  to the same assembly as tracks

### DISCUSSION THREAD

**Discussion by Vishvak2000** [GMOD/jbrowse-components#2211] [Q&A] [answered]
I am trying to configure JBrowse2 for a viral proteome (assemblies for each of the viral proteins). This virus has many genotypes and I would like to have tracks that show the consensus sequence for each of the genotypes. Is there a way to add another sequence (in fasta/indexed fasta format) as a track to an assembly?

**Comment by cmdcolin** [comment, marked as answer]
It would probably not work for jbrowse to have multiple "reference sequence tracks" displayed at the same time. If you wanted to make a track type that has the strains compared with the reference there are maybe a couple options

1) In jbrowse 1 here we have an example of displaying a "MAF" file (multiple alignment format) which is commonly used for whole genome alignments https://github.com/cmdcolin/mafviewer no equivalent exists for jbrowse 2 yet currently though

2) Alternatively multiple sequence aligners are probably the most common way to compare proteomes (there is less of a "reference bias" with a multiple sequence alignment since no particular sequence is necessarily a reference). We are currently working on one of these here, and it will also have some jbrowse 2 intergration in the future (it can already be used as a jbrowse 2 plugin)  https://gmod.github.io/react-msaview/


3) Another option and maybe the most straightforward in JBrowse 2 would be to make a VCF comparing the viral strains versus a reference strain, so you only see the differences as VCF features on a normal feature type track
